src/glob.zig

```diff
@@ -0,0 +1,2555 @@
+const std = @import("std");
+const math = std.math;
+const mem = std.mem;
+const BunString = @import("./bun.zig").String;
+const expect = std.testing.expect;
+const Allocator = std.mem.Allocator;
+const ArrayList = std.ArrayListUnmanaged;
+const ArrayListManaged = std.ArrayList;
+const DirIterator = @import("./bun.js/node/dir_iterator.zig");
+const Syscall = @import("./sys.zig");
```

Comment:
This should really be encoded in a linter, but can you use namespaces instead of relative imports when possible? This makes it easier for us to move files around in the future and sometimes also with autocomplete 

`bun.sys` in this case